### PR TITLE
docs: 📚 Document used typographic styles and fonts

### LINKED ISSUE #614: docs: 📚 Storybook and demos do not use the correct open-sans font
# Description

We recently got the question if and how `Open Sans` is provided. The user had a look at our storybook instance and could not find it or information regarding font handling in general. After further investigation, it seems we may have forgotten to load our own brand font in Storybook and our Framework demos.

# Rationale

As a developer of the Synergy Design System, I want to make sure our demos are aligned with our styles.

# Acceptance criteria

- [ ] Storybook loads `Open Sans` and all components use it.
- [ ] Angular Demo loads `Open Sans` and all components use it.
- [ ] React  Demo loads `Open Sans` and all components use it.
- [ ] Vanilla Demo loads `Open Sans` and all components use it.
- [ ] Vue Demo loads `Open Sans` and all components use it.
- [ ] Unit and Integration Tests load `Open Sans` and all components use it.
- [ ] A follow up documentation ticket is created where we tackle a better documentation for our users.

### LINKED ISSUE #615: docs: 📚 Document used typographic styles and fonts
# Description

Synergy should provide documentation about the required font setup for projects using it. As can be seen in #614, this may lead to problems (because developers forget to load the font).

Therefore, we have to provide additional documentation in Storybook.

# Rationale

As a developer using the Synergy Design System, I want to have good documentation that helps me to get started using Synergy with all required steps.

# Acceptance criteria

- [ ] A new chapter for "Getting Started" is created in the [components README.md file](https://github.com/synergy-design-system/synergy-design-system/blob/main/packages/components/README.md)
- [ ] The chapter must provide a link to the [Open Sans WebFont used in the Brand Portal](https://brand.sick.com/r/uZVZ3fiEbHTDRkDCuFueNDu_9GeMRqj0cMs46tCIqqg)
- [ ] A new FAQ entry is created explaining why Synergy does not provide the font out of the box and shows possible ways of loading.

### PR DESCRIPTION
**Pull request by schilchSICKAG**
# Pull Request

## 📖 Description

This PR adds `Open Sans` as a dependency to our demo (via external npm package) and makes sure the font is also used in Storybook (using the officially supported archive from SDS).

It also adds documentation as described in #615.

### 🎫 Issues

Closes #614
Closes #615 

## 👩‍💻 Reviewer Notes

Just play around with Storybook and the framework demos (e.g. inspect loading via Network-Panel, check if correct font is used, check dist versions).

## 📑 Test Plan

- Check out this branch locally (mandatory!)
- Run `pnpm i -r` (or `pnpm clean` in the repository root)
- Start Storybook. Notice the new font in contrast to the current one.
- Check that all components use the correct font in Storybook
- Move to `packages/_private` and inspect the same for the framework demos
- Let the tests start via `cd packages/_private/e2e-demo-test && pnpm start`. There should not be any errors.

## ✅ DoD

- [x] I have tested my changes manually.
- [x] I have added automatic tests for my changes (unit- and visual regression tests).
- [x] I have updated the project documentation to reflect my changes (e.g. CHANGELOG, Storybook, ...).
- [ ] ~~I have added documentation to the DaVinci migration guide (if need be)~~
- [x] I have made sure to follow the projects coding and contribution guides.
- [x] I have made sure that the bundle size has changed appropriately.
- [x] I have validated that there are no accessibility errors.
- [x] I have used design tokens instead of fix css values

### CODE REVIEW

**Review by kirchsuSICKAG — CHANGES_REQUESTED** [review]

File: packages/docs/.storybook/main.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ const config: StorybookConfig = {
     "../stories/Welcome.mdx",
     "../stories/GoalsAndGuidelines.mdx",
     "../stories/Presentation.mdx",
+    "../stories/Prequisites.mdx",
```
[the remark this answers is not in the document]
Comment (schilchSICKAG) [reply packages/docs/.storybook/main.ts]: Nice catch, just updated it.

File: packages/docs/stories/Prerequisites.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+<Meta title="Prerequisites" />
+
+# Prerequisites
+
+## Fonts
+
+The Synergy Design System makes use of the `Open Sans` typeface.
+Because there are various ways of loading fonts, depending on used bundler, pre- and postprocessors and usage of a CDN, Synergy does **not provide** this font.
+You will need to include it in your project.
```
Comment (klaarseSICKAG) [inline packages/docs/stories/Prerequisites.mdx:12]:
```suggestion
You need to include it into your own project.
```

File: packages/docs/stories/Prerequisites.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+The following information may be helpful to get you started including your own copy of `Open Sans` in your project for quicker bootstrapping:
+
+### Local Installation
+
+1. Download the `Open Sans` font from the [SICK Brand Portal](https://brand.sick.com/document/49#/-/resources-1).
```
Comment (klaarseSICKAG) [inline packages/docs/stories/Prerequisites.mdx:18]: Why not referencing the fonts on GitHub (packages/docs/public/OpenSans) ?
I'd prefer not to reference servers we don't have control over.

File: packages/docs/stories/Prerequisites.mdx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+2. Extract the ZIP file to a destination reachable by your project (e.g. a `public` folder).
```
Comment (klaarseSICKAG) [inline packages/docs/stories/Prerequisites.mdx:19]: this step can be skipped then

File: packages/docs/stories/Prerequisites.mdx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+### Installing via Google Fonts
+
+> ⚠️ This is only suited for development purposes and **must never be used in production**.
```
Comment (klaarseSICKAG) [inline packages/docs/stories/Prerequisites.mdx:116]: Could we add the "why" ? Why is only suitable for development? Why should I use this code in the first place?

File: packages/tokens/README.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # @synergy-design-system/tokens
 
 This package provides the design tokens that form the base for all components of the synergy design system.
```
Comment (klaarseSICKAG) [inline packages/tokens/README.md:3]:
```suggestion
This package provides the design tokens that form the base for all components of the Synergy Design System.
```

File: packages/docs/stories/Prerequisites.mdx (hunk 3 above)
Comment (schilchSICKAG) [reply packages/docs/stories/Prerequisites.mdx:18]: As the brand portal is the single source of truth, we will leave the comment as discussed.

File: packages/docs/stories/Prerequisites.mdx (hunk 5 above)
Comment (schilchSICKAG) [reply packages/docs/stories/Prerequisites.mdx:116]: see above

File: packages/docs/stories/Prerequisites.mdx [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+### Using the SICK CDN
+
+This is the simplest default font behavor. The `Open Sans` font is loaded from the SICK CDN.
```
Comment (kirchsuSICKAG) [inline packages/docs/stories/Prerequisites.mdx:116]: typo `behavor` --> `behavior`

```suggestion
This is the simplest default font behavior. The `Open Sans` font is loaded from the SICK CDN.
```

File: packages/docs/stories/Prerequisites.mdx (hunk 3 above)
Comment (reichst1SICKAG) [reply packages/docs/stories/Prerequisites.mdx:18]: Google also only provides .ttf files in the official GitHub

Comment (schilchSICKAG) [reply packages/docs/stories/Prerequisites.mdx:18]: Another valid point. I think there was the possibility to load the `woff` files somewhere, but I did not find it yesterday, too.

**Review by reichst1SICKAG — APPROVED** [review]
Do we have italic used somwhere?
We don't provide woff files at brand.sick.com and I can't remember that we use it in any component.

File: packages/docs/stories/Prerequisites.mdx (hunk 3 above)
Comment (reichst1SICKAG) [reply packages/docs/stories/Prerequisites.mdx:18]: Yes, I also can't remember where I have downloaded the woff files initially.
Maybe buried somwhere in the older version of Google Fonts.

**Pull request merged**